config/locales/en.yml

```diff
@@ -18,3 +19,6 @@ en:
     assignee set, tick the box below to assign the issue to the user who just
     created the issue, if possible. If the box is left unticked, there will be
     no assignee, which is the default behaviour in Redmine.
+  default_assign:
+    override_category_assignee_info: "Override Category Assignee"
+    override_category_assignee: "Override Category Assignee"
```

Comment:
Can you add these keys to the other translation files as well? A copy of the English versions will do fine until someone submits a translation.